srcpkgs/apache-kafka/template

```diff
@@ -1,20 +1,19 @@
 # Template file for 'apache-kafka'
 pkgname=apache-kafka
-version=2.3.0
+version=2.4.0
 revision=1
 wrksrc="kafka-${version}-src"
-hostmakedepends="gradle"
+hostmakedepends="curl which tar openjdk8"
```

Comment:
Not sure about `openjdk8`.

Without it, the build says it cannot find java on PATH.

I looked into it briefly, and it looked like `depends="virtual?java-runtime"` doesn't install the java alternative, so maybe it's not linked into `/usr/bin`?

Adding `openjdk8` to `hostmakedepends` fixes it and should allow it to run on any JRE we have (8 or 11)